prebid-server/endpoints/openrtb2/amp.md

```diff
@@ -95,8 +101,43 @@ A sample response payload looks like this:
 In [the typical AMP setup]({{site.baseurl}}/dev-docs/show-prebid-ads-on-amp-pages.html),
 these targeting params will be sent to DFP.
 
-Note that "errors" will only appear if there were any errors generated. They are identical to the "errors" field in the response.ext of the OpenRTB endpoint.
+### Error Messages
+
+If any errors were generated they will appear  within `response.ext.errors.{bidderName}`. There are five error codes that could be returned: 
+
+```
+0   NoErrorCode
+1   TimeoutCode
+2   BadInputCode
+3   BadServerResponseCode
+999 UnknownErrorCode
+```
 
+See the [/openrtb2/auction endpoint](/prebid-server/endpoints/openrtb2/auction.html) for a description of some common openRTB errors. The following is a list of AMP specific errors that could be returned: 
+
+{: .table .table-bordered .table-striped }
+| Task  | Code |  Message | Action  |
+|---|---|---|---|
+|  Returning auction data. | 3  | Critical error.  | status set to `StatusInternalServerError`.  |
+|  Extracting the targeting parameters from the response. | 3  | Critical error while unpacking AMP targets.  | status set to `StatusInternalServerError`.  |
+|  Extract error from response. | 999  | AMP response: failed to unpack OpenRTB response.ext, debug info cannot be forwarded.  | Error is logged.  |
+|  Adding debug information. | 999  | Test set on request but debug not present in response.  | Error is logged.  |
+|  Encoding the response. | 999  | `/openrtb2/amp` failed to send response.  | Error is logged.  |
+|   |   |   |   |
+
+The following errors can occur when loading a stored OpenRTB request for an incoming AMP request.
+
+{: .table .table-bordered .table-striped }
+| Task  | Code |  Message | Action  |
+|---|---|---|---|
+|  Query check for tag_id. | 999  | AMP requests require an AMP tag_id.  | Error is returned.  |
+|  Checking stored request for match against tag_id. | 999  | No AMP config found for tag_id `%s`.  | Error is returned.  |
+|  Checking if imp exists. | 999  | Data for tag_id=`'%s'` does not define the required imp array.  | Error is returned.  |
+|  Checking if imp count is greater than one. | 999  | Data for tag_id `'%s'` includes `%d` imp elements. Only one is allowed.  | Error is returned.  |
+|  Checking if request.app exists. | 999  | `r`equest.app` must not exist in AMP stored requests.  | Error is returned.  |
```

Comment:
check code formatting round request.app, there's an extra backtick in there.

Reply:
Fixed.